lib/ansible/module_utils/k8s/common.py

```diff
@@ -167,9 +167,6 @@ def execute_module(self):
 
         return_attributes = dict(changed=False, result=dict())
 
-        if self._diff:
```

Comment:
This is bad.

There is an if condition here, controlled by the user.

Why remove this option from the user?